pages/queryKeys.ts

```diff
@@ -1,5 +1,8 @@
+//descriptive names: Something like 'GET_COMMENTS', 'GET_COMMENT', 'ADD_USER'
+
 const queryKeys = {
-  COMMENT: "comment",
+  GET_COMMENT: "get_comment",
+  GET_USER: "get_user",
```

Comment:
I think these should be all caps, the values. Because they are constants, and in JS world, we use CAPS_CAMEL_CASE to represent constants